recipes/crypto-rsa-pss-update/tests/expected/promisified-wrappers.js

```diff
@@ -0,0 +1,46 @@
+const crypto = require('crypto');
+const util = require('util');
+
+// Case 1: Basic promisified wrapper
+const generateKeyPairAsync = util.promisify(crypto.generateKeyPair);
```

Comment:
Create other test with variantion of promisify import

```suggestion
const {promisify} = require('util');

// Case 1: Basic promisified wrapper
const generateKeyPairAsync = promisify(crypto.generateKeyPair);
```